app/controllers/concerns/search_menus.rb

```diff
@@ -0,0 +1,14 @@
+module SearchMenus
```

Comment:
I don;t think this is a good name. Maybe MenuDataPreload or somthing like that

Reply:
done